From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Gerald Elder-Vass <47088217+GeraldEV@users.noreply.github.com>
Date: Wed, 8 Nov 2023 03:44:22 +0000
Subject: [PATCH] Move Surge Alloy in ErekirTechTree to the correct position
 (#9253)

Surge Alloy can be first obtained on the "Ravine" map.
Thorium can be first obtained on the "Caldera" map.
The "Ravine" map must be unlocked and completed before the "Caldera" map can be accessed.
Since Surge Alloy isn't required to produce Thorium (and vice versa), they should appear at the same level of the tech tree, under Tungsten.

Signed-off-by: Gerald Elder-Vass <geraldev@hotmail.co.uk>
(cherry picked from commit 498bf079ec152dae273cd1cff3acfa4a160d064b)
---
 core/src/mindustry/content/ErekirTechTree.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/src/mindustry/content/ErekirTechTree.java b/core/src/mindustry/content/ErekirTechTree.java
index 89f128ae725869562f64af7deaedb19fbd556cfe..6c4571a43ad8bdd2ed98dc72a996bee6c2fc7af2 100644
--- a/core/src/mindustry/content/ErekirTechTree.java
+++ b/core/src/mindustry/content/ErekirTechTree.java
@@ -447,11 +447,11 @@ public class ErekirTechTree{
 
                                 //nodeProduce(Liquids.gallium, () -> {});
                             });
+                        });
 
-                            nodeProduce(Items.surgeAlloy, () -> {
-                                nodeProduce(Items.phaseFabric, () -> {
+                        nodeProduce(Items.surgeAlloy, () -> {
+                            nodeProduce(Items.phaseFabric, () -> {
 
-                                });
                             });
                         });
                     });
